frontend/src/App.tsx

```diff
@@ -1,12 +1,13 @@
 import { Outlet, Route, Routes } from "react-router-dom";
 
 import { NavBar } from "./components/index.ts";
-import { Apply, Candidates } from "./pages/index.ts";
+import { Home, Apply, Candidates } from "./pages/index.ts";
 
 function Layout() {
   return (
     <>
       <NavBar></NavBar>
```

Comment:
Actually can we also change this to <NavBar /> since there is no need to have it wrap anything